cmd/operator-sdk/test/local.go

```diff
@@ -293,6 +243,162 @@ func testLocalGoFunc(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func collectManifests() error {
+	// Pass empty files to the test-framework if no setup required
+	if tlConfig.noSetup {
+		emptyBytes := []byte{}
+		file, err := writeToTempFile(emptyBytes, "empty.yaml")
+		if err != nil {
+			return err
+		}
+		tlConfig.namespacedManPath = file.Name()
+		tlConfig.globalManPath = file.Name()
+
+		f := func() {
+			err := os.Remove(file.Name())
+			if err != nil {
+				log.Errorf("Could not delete empty manifest file: (%v)", err)
+			}
+		}
+		cleanups = append(cleanups, f)
+		return nil
+	}
+
+	// Collect manifest from deploy dir if legacy layout
+	// TODO: Remove this after the legacy layout has been deprecated
+	if !kbutil.HasProjectFile() {
+		return legacyCollectFromDeployDir()
+	}
+
+	// For new layout check if the manifests are piped to stdin
+	stdinF, err := os.Stdin.Stat()
+	if err != nil {
+		return err
+	}
+	if stdinF.Mode()&os.ModeNamedPipe != 0 {
+		return collectFromStdin()
+	}
+
+	// If no input is piped to stdin, the manifest paths must be set by flags
+	if tlConfig.namespacedManPath == "" || tlConfig.globalManPath == "" {
+		return fmt.Errorf("The namespaced and global manifests must be piped to stdin" +
+			" e.g (kustomize build config/default | operator-sdk test ...).\n" +
+			"Or the both the flags --namespaced-manifest and --global-manifest must be set.")
+	}
```

Comment:
I am not sure if I got what it does and the steps to solve. 

```suggestion
	// If no input is piped to stdin, the manifest paths must be set by flags
	if tlConfig.namespacedManPath == "" || tlConfig.globalManPath == "" {
		return fmt.Errorf("Unable to identify thee namespaced and global manifests." +
			"Use the flags --namespaced-manifest and --global-manifest to set or generating the manifest. E.g kustomize build config/default")
	}
	return nil
```

Why the solution would be kustomize build config/default? Have we not an sdk command to do generate the manifest? Would not it be make manifest as well?